Remove hacky double registration of to_here op in reg_distributed_ops

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#39602 Remove hacky double registration of to_here op in reg_distributed_ops**

This was added as a part of
https://github.com/pytorch/pytorch/pull/38590 but we can use default arguments
here. We use fmt:;format to bind the default value to the rpc timeout at
runtime.

Differential Revision: [D21912719](https://our.internmc.facebook.com/intern/diff/D21912719/)

rohan-varma committed:
Remove hacky double registration of to_here op in reg_distributed_ops

This was added as a part of
https://github.com/pytorch/pytorch/pull/38590 but we can use default arguments
here. We use fmt:;format to bind the default value to the rpc timeout at
runtime.

Differential Revision: [D21912719](https://our.internmc.facebook.com/intern/diff/D21912719/)

[ghstack-poisoned]

rohan-varma committed:
Update on "Remove hacky double registration of to_here op in reg_distributed_ops"


This was added as a part of
https://github.com/pytorch/pytorch/pull/38590 but we can use default arguments
here. We use fmt:;format to bind the default value to the rpc timeout at
runtime.

Differential Revision: [D21912719](https://our.internmc.facebook.com/intern/diff/D21912719/)

[ghstack-poisoned]

rohan-varma committed:
Update on "Remove hacky double registration of to_here op in reg_distributed_ops"


This was added as a part of
https://github.com/pytorch/pytorch/pull/38590 but we can use default arguments
here. We use fmt:;format to bind the default value to the rpc timeout at
runtime.

Differential Revision: [D21912719](https://our.internmc.facebook.com/intern/diff/D21912719/)

[ghstack-poisoned]

rohan-varma committed:
Update on "Remove hacky double registration of to_here op in reg_distributed_ops"


This was added as a part of
https://github.com/pytorch/pytorch/pull/38590 but we can use default arguments
here. We use fmt:;format to bind the default value to the rpc timeout at
runtime.

Differential Revision: [D21912719](https://our.internmc.facebook.com/intern/diff/D21912719/)

[ghstack-poisoned]

rohan-varma committed:
Update on "Remove hacky double registration of to_here op in reg_distributed_ops"


This was added as a part of
https://github.com/pytorch/pytorch/pull/38590 but we can use default arguments
here. We use fmt:;format to bind the default value to the rpc timeout at
runtime.

Differential Revision: [D21912719](https://our.internmc.facebook.com/intern/diff/D21912719/)

[ghstack-poisoned]

xush6528 commented on torch/csrc/jit/runtime/register_distributed_ops.cpp:
```diff
@@ -26,24 +28,9 @@ static auto workerInfo =
-     Operator(
-         "aten::to_here(RRef(t) self, double timeout) -> t(*)",
+         fmt::format(
+             "aten::to_here(RRef(t) self, float timeout = {}) -> t",
```
"aten::to_here(RRef(t) self, float timeout = {}) -> t(*)",

rohan-varma replied:
@xush6528 fixed.

rohan-varma committed:
Update on "Remove hacky double registration of to_here op in reg_distributed_ops"


This was added as a part of
https://github.com/pytorch/pytorch/pull/38590 but we can use default arguments
here. We use fmt:;format to bind the default value to the rpc timeout at
runtime.

Differential Revision: [D21912719](https://our.internmc.facebook.com/intern/diff/D21912719/)

[ghstack-poisoned]
